### ISSUE TITLE
After 2.14 $state.go won't work for route with queries

### ISSUE THREAD

**Issue by jptannus** [angular-ui/ui-router#1937]
Hi I have the following state:
```
.state('main.lists.list.detail', {
          url: '/{listId:[0-9]{1,1000}}?expanded',
          templateUrl: 'app/lists/lists-list/lists-list-detail.html',
          controller: 'ListsListDetailCtrl',
          resolve: {
            showPanel: ['$stateParams', function($stateParams) {
              $stateParams.showPanel = true;
            }]
          }
        })
```
And in a part of my code I got this:
```
$state.go('main.lists.list.detail', params);
```

In version 2.13 it all works fine, but when I updated it wasn't working anymore. I removed the `?expanded` from the state definition and it worked fine.

So are query values like this not working anymore?

Thanks

**Comment by akullpp** [comment]
Experiencing the same issue.

**Comment by Cheema1** [comment]
did you define $state.go

**Comment by jptannus** [comment]
Define $state.go? I'm not getting any errors. It is just not working.

I use $state go for other states and as I said if I remove the `?expanded` it works.

**Comment by eddiemonge** [comment]
dupe of #1915

**Comment by eddiemonge** [comment]
Please check the fix in #1915 and if it doesn't work for you report back and I'll reopen this. Thanks
